docs/source/monitoring/index.rst

```diff
@@ -0,0 +1,11 @@
+Monitoring
+===================
+
+This section covers details on monitoring the state of your JupyterHub installation.
+
+Jupyterhub operational metrics are collected using Prometheus. For more details about Prometheus please refer to the [official documentation](https://prometheus.io/docs/introduction/overview).
```

Comment:
I love a good tech intro to those not yet having an overview of the situation, like myself a year ago, so I made an attempt to provide one!

```suggestion
JupyterHub expose the ``/metrics`` endpoint that returns text describing its current operational state formatted in a way `Prometheus <https://prometheus.io/docs/introduction/overview/>_` understands. Prometheus is a separate open source tool that can be configured to repeatedly poll JupyterHub's ``/metrics`` endpoint to parse and save the momentary state. By doing so, Prometheus can describe JupyterHub's eolving state over time. This evolving state can then be accessed through Prometheus that expose its underlying storage to those allowed to access it, and be presented with dashboards by a tool like `Grafana <https://grafana.com/docs/grafana/latest/getting-started/what-is-grafana/>_`.
```